bagel/tests/test_utility.py

```diff
@@ -8,15 +7,21 @@
 from bids import BIDSLayout
 
 import bagel.bids_utils as butil
+import bagel.derivatives_utils as dutil
+import bagel.file_utils as futil
 import bagel.pheno_utils as putil
-from bagel import mappings
-from bagel.utility import load_json
+from bagel import mappings, models
+from bagel.utility import (
+    extract_subs_from_jsonld_dataset,
+    generate_context,
+    get_subs_missing_from_pheno_data,
+)
 
```

Comment:
very minor but why do we namespace import all of the other util modules, but expand the `bagel.utility` imports?

Reply:
Hmm, no real reason I think, maybe just as another way to differentiate command-specific vs. common utilities when looking at the code. There tends to be fewer shared utilities, so it wasn't much additional effort to import them. That said, it probably doesn't hurt to make this consistent with the other modules. Could address in #350